### ISSUE TITLE
Saving plots doesnt work on windows.

### ISSUE THREAD

**Issue by arhik** [lfiondella/SRT#26]
Windows platform needs to be fixed

**Comment by arhik** [comment]
We should check this on windows. If any windows user had a chance to check this please let me know how it works.

**Comment by MilesWT** [comment]
I will look into this issue as well, as it indeed does not work and instead the save display button opens a new tab of the site. Tested in Chrome and Firefox.

**Comment by a1729nikora** [comment]
I looked into this a couple of days ago on a 64-bit Windows 7 system after first hearing about it.  When running SRT from within the RStudio window, I found similar behavior - I could save plots as pdf, but in no other formats.  However, when I ran it in Chrome, I was able to save plots in all formats with the proper scaling.  I'll take another look when I get back to the office and can get at the Windows system I ran it on.

-Allen N.

# 

   Allen P. Nikora, Ph.D.
   Program Element Manager,
        Assurance Technology Program Office Software Element

   Jet Propulsion Laboratory
   California Institute of Technology
   4800 Oak Grove Drive
   Pasadena, CA  91109-8099
   Mail Stop 125-233

   vox: 818.393.1104
   fax: 818.393.1362
   email: Allen.P.Nikora@jpl.nasa.gov

#

**Comment by arhik** [comment]
This should be addressed soon. It makes sense to assign it to one of the under graduates who is working on windows platform. If any one could do it, please let me know.

**Comment by arhik** [comment]
This is tied to basic functionality milestone. We should have made three seperate milestones for windows, mac and linux.
